Demo/Demo/FeatureViewControllers/PayPalDemoViewController.swift

```diff
@@ -31,8 +32,13 @@ class PayPalDemoViewController: FeatureBaseViewController {
         NSLayoutConstraint.activate([
             payPalButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
             payPalButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Constants.layoutSpacing),
-            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
-            payPalButton.heightAnchor.constraint(equalToConstant: Constants.textFieldHeight)
+            payPalButton.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: 50),
+            payPalButton.heightAnchor.constraint(equalToConstant: Constants.textFieldHeight),
+
+            payPalCreditButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.layoutSpacing),
```

Comment:
Do we have an UIView extension to make more erase to do this kind of setup for constraints? 

Example:

```
extension UIView {
  func anchor(to view: UIView, ...) {}
}
```

Reply:
Also, keeping our demo app straightforward per-module is nice.